source/utils/path.cpp

```diff
@@ -231,8 +229,20 @@ const std::string &path::string() const
 #ifdef _MSC_VER
 std::wstring path::wstring() const
 {
-    std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> convert;
-    return convert.from_bytes(string());
+    const std::string &path_str = string();
+    int size = MultiByteToWideChar(CP_UTF8, MB_ERR_INVALID_CHARS, path_str.c_str(), static_cast<int>(path_str.length()), nullptr, 0);
```

Comment:
Note sure if it is better to use `MB_ERR_INVALID_CHARS` or 0 as second arg.

Reply:
[This StackOverflow answer](https://stackoverflow.com/a/22845967) agrees with me from a security perspective. I can also say that I use `MultiByteToWideChar` like this since multiple years in some projects, and for me it worked perfectly. I never tried to use 0, so I cannot say anything against it - except that the StackOverflow post recommends using `MB_ERR_INVALID_CHARS` as well :slightly_smiling_face:

Reply:
Ok